### PR TITLE
feat(readme): import readme package

### PR DESCRIPTION
**Pull request by sergiohgz**
**Type:** feat
**Scope:** readme

The following has been addressed in the PR:

*  There is a related issue? **Part of #23**
*  Unit or Functional tests are included in the PR **No**

**Description:** Import readme package in core/readme and modernize project.
_Note:_ Paraphrasing [npm docs](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package):
> If, given the structure of your project, you find .npmignore to be a maintenance headache, you might instead try populating the files property of package.json, which is an array of file or directory names that should be included in your package. Sometimes a whitelist is easier to manage than a blacklist."

I prefer to whitelist `lib/` folder in `package.json` instead make a blacklist that need more maintenance

### CODE REVIEW

**Comment by sergiohgz** [comment]
The last commit added is needed to insert the definitions created with `build:types` script in the package

File: core/readme/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+# 2 space indentation
+[{.,}*.{js,yml,yaml}]
```
Comment (juanpicado) [inline core/readme/.editorconfig]: Perhaps add here `ts` or is too soon?

Comment (juanpicado) [reply core/readme/.editorconfig]: Ah, it is an import, nevermind.

**Comment by sergiohgz** [comment]
Sorry @juanpicado , I've just dismissed your review when removing old and unnecessary scripts, my fault

**Comment by juanpicado** [comment]
Done :-)

**Pull request merged**